Jenkinsfile

```diff
@@ -95,6 +95,7 @@ pipeline {
         branch 'master'
       }
       steps {
+        checkPactCompatibility("adminusers", gitCommit(), "test")
```

Comment:
https://github.com/alphagov/pay-jenkins-library/blob/master/vars/checkPactCompatibility.groovy requires the `consumer` and `consumerVersion` , but you are passing in the provider. Bit confusing

Reply:
ah. it's actually checkPactCompatibility that needs fixing. it's arguments should be `service` and `version`, not `consumerName` and `consumerVersion` as it doesn't matter whether you pass in the consumer or provider. i'll raise a PR for that.